### ISSUE TITLE
Does this tool work with async loaded data yet??

### ISSUE THREAD

**Issue by twaldron** [valor-software/ng2-select#506]
I am loading the data I am trying to bind do from a web service. The data is loaded in the subscribe function of the observable. The object has the id and text properties, but nothing shows up in the select when the data is loaded. If i hard code some values and not fetch from the webservice they appear.

**Comment by usmanwalana** [comment]
Same Issue with me. The console shows my fetched data but nothing in the select.

**Comment by twaldron** [comment]
Yeah, I have tried all the work arounds and hacks mentioned in other discussions and none seem to work with current versions.

**Comment by RodolfoSilva** [comment]
Guys, you need convert the data to this format:

```
[
   {id: ..., text: ... }
]
```

**Comment by RodolfoSilva** [comment]
https://github.com/valor-software/ng2-select/issues/97#issuecomment-224582992

**Comment by JimJafar** [comment]
This might help: https://github.com/valor-software/ng2-select/issues/635#issuecomment-281094377
